Assets/unZENity-VR/Scripts/Settings/SettingsManager.cs

```diff
@@ -65,5 +72,28 @@ private void CheckIfGothicIDirectoryExists()
                     $"Please put in the right absolute path to your local installation.");
         }
 
+        /// <summary>
+        /// Import the settings file from streamingAssetPath to persistentDataPath.
+        /// Since the settings file is in streamingAssetPath, we need to use UnityWebRequest to move it so we can have access to it
+        /// as detailed here https://docs.unity3d.com/ScriptReference/Application-streamingAssetsPath.html
+        /// </summary>
+        private void Importer()
+        {
+            string GameSettingsPath = System.IO.Path.Combine(Application.streamingAssetsPath, $"{SETTINGS_FILE_NAME}");
+            string result = "";
+            if (GameSettingsPath.Contains("://") || GameSettingsPath.Contains(":///"))
+            {
+                UnityEngine.Networking.UnityWebRequest www = UnityEngine.Networking.UnityWebRequest.Get(GameSettingsPath);
```

Comment:
Do we really need WebRequest? Can't we just leverage default System.Io.File.Copy() or sth. like that?
(I don't know if this can be used to copy data from within an APK into the local file system, but we should test.)

Reply:
`Android uses a compressed .apk file. These platforms return a URL. Use the [UnityWebRequest](https://docs.unity3d.com/ScriptReference/Networking.UnityWebRequest.html) class to access the Assets.` this is a quote directly from https://docs.unity3d.com/ScriptReference/Application-streamingAssetsPath.html